### PR TITLE
WCMP2 ID s/reco/recommended/g

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

**Comment by tomkralidis** [comment]
Correct, there is no requirement for the "local identifier" token of a WCMP2 id.  At the same time, in the wis2box context there is an inconsistency ("core -> core", "recommended" -> "reco"), driven by a user interface layout.

I'm fine if we can discuss a consistent approach to "local" identifiers in wis2box without morphing official values into conventions driven by a UI.

File: src/components/DatasetEditorForm.vue [hunk 1]
```diff
@@ -1389,20 +1389,20 @@ export default defineComponent({
             // Truncate policy to 4 letters
             let truncatedPolicy = model.value.identification.wmoDataPolicy.substring(0, 4);
```
[the remark this answers is not in the document]
Comment (tomkralidis) [reply src/components/DatasetEditorForm.vue]: Fixed.

**Comment by maaikelimper** [comment]
https://private-user-images.githubusercontent.com/48341438/398385672-50bb9803-ce99-44fa-8040-3a705b1548fd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwMzU1NTEsIm5iZiI6MTczNTAzNTI1MSwicGF0aCI6Ii80ODM0MTQzOC8zOTgzODU2NzItNTBiYjk4MDMtY2U5OS00NGZhLTgwNDAtM2E3MDViMTU0OGZkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI0VDEwMTQxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJjZDJhNDYyMTdkMzgxYThlYWZjNGZkYjM2MGM3NWViZjY4OWI0OTJlYjdhMDNhYjIxNjgxMzcyYTUyMDMyZGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ulzFcgc4lkcHnKell8OBSld73H4PHXtZuFDUUfqpmm0

**Comment by maaikelimper** [comment]
@tomkralidis this PR can be closed as the changed introduced by the latest PR remove the use of data-policy and instead the user is invited to provide a "local ID" 
https://github.com/wmo-im/wis2box-webapp/pull/79

**Pull request closed without merging**